hardware/trackball.scad

```diff
@@ -1,6 +1,6 @@
 
 // This module prints the full trackball body, with all cutouts.
-// full();
+full();
```

Comment:
I recently removed this line, so that the .scad files in the stl directory can include trackball.stl and use selected parts of it.